iOS_SDK/OneSignalSDK/Source/OneSignalHelper.m

```diff
@@ -785,21 +790,16 @@ + (void)addNotificationRequest:(OSNotificationPayload*)payload
 + (NSString*)downloadMediaAndSaveInBundle:(NSString*)urlString {
```

Comment:
`+ (NSString*)downloadMediaAndSaveInBundle:(NSString*)urlString` 
needs spacing 
`+ (NSString *)downloadMediaAndSaveInBundle:(NSString *)urlString`

Reply:
One space was forgotten here, resolving and added another comment